ui/ui-components/scss/_variables.scss

```diff
@@ -0,0 +1,40 @@
+// Grouparoo Colors
+
+$grouparoo-light-gray: #adb5bd;
+$grouparoo-dark-gray: #242436;
+$grouparoo-yellow: #ff7518;
+$grouparoo-light-yellow: #fff8f1;
+$grouparoo-red: #ff0039;
+$grouparoo-blue: #29abe2;
+$grouparoo-mint: #2fe5a4;
+$grouparoo-light-blue: #b3e8ff;
+$grouparoo-blue-gray: #9b9db1;
+$grouparoo-violet: #9954bb;
+$grouparoo-light-violet: #dfe8fb;
+$grouparoo-sky-blue: #6090ed;
+$grouparoo-spice: #edac60;
+$grouparoo-light-violet: #dfe8fb;
+$grouparoo-background-blue: #f6fafb;
+$grouparoo-green: #3fb618;
+$grouparoo-solid: #3d3d3d;
+$grouparoo-ethereal-blue: #f5f9ff;
+
+// Bootstrap Colors, cannot be CSS variables
+$primary: $grouparoo-blue;
+$secondary: $grouparoo-light-blue;
+$success: $grouparoo-mint;
+$info: $grouparoo-sky-blue;
+$warning: $grouparoo-yellow;
+$danger: $grouparoo-red;
+$light: $grouparoo-light-gray;
+$dark: $grouparoo-dark-gray;
```

Comment:
There are some differences from the www styles here:
- danger is red, rather than orange
- secondary is light blue, rather than mint
- success is mint, rather than another green